### ISSUE TITLE
Double data in Chrome

### ISSUE THREAD

**Issue by lembitk** [keithclark/gadebugger#31]
- Chrome: 49.0.2623.110 m
- Google Analytics Debugger v2.1.0

Somehow all data (pageviews, events) is double-logged, please check the screenshot: 

![gadebugger-chrome-double-data](https://cloud.githubusercontent.com/assets/1476817/14228944/cea81316-f92e-11e5-941f-66328a18e765.png)

I'm sure it's not the fault of my tracking code implementation or a problem with my event tracking JS, because, for one, double data  doesn't occur in Firefox devtools (Google Analytics Debugger v2.1.1). Also, Chrome's [Google Tag Assistant](https://chrome.google.com/webstore/detail/tag-assistant-by-google/kejbdjndbnbjgmefkgdddjlbokphdefk) reports "1 Pageview Requests", "1 Events" every time when GA Debugger reports 2.

**Comment by keithclark** [comment]
Thanks for the bug report.

GA Debugger watches for HTTP(S) requests to the GA domain and logs a beacon for each one. It looks like GA is now forcing all HTTP beacon requests to be re-transmitted over HTTPS, this is why you're seeing two requests in the extension.

**Request Headers:**
```
Request URL:http://www.google-analytics.com/collect?...
Request Method:GET
Status Code:307 Internal Redirect
```

**Response Headers:**
```
Location:https://www.google-analytics.com/collect?
Non-Authoritative-Reason:HSTS
```

**Comment by lembitk** [comment]
Thanks for the answer, and thank you for making such a handy tool.

**Comment by keithclark** [comment]
#32 should fix the issue.

**Comment by keithclark** [comment]
Bug fixed and deployed in v2.1.2.
